highspy/meson.build

```diff
@@ -0,0 +1,20 @@
+py_mod = import('python')
+py = py_mod.find_installation(pure: false)
+pyb11_dep = [
+  py.dependency(),
+  dependency('pybind11')
+]
+_deps += pyb11_dep
```

Comment:
Due to ordering, this file is processed last in the build -- so `_deps` is not referred to again, after this, I think.

But if it were, then this seems to be a global array and would inject the py installation dependency as well as pybind11 into other targets, right?

I would probably have done something like:
```meson
pyb11_dep = [
  _deps,
  py.dependency(),
  dependency('pybind11')
]
```

Reply:
Yup, I reworked the logic a bit to make it cleaner (hopefully).